### ISSUE TITLE
frequency='7d' data.history(..) resampling not working in live mode with Bittrex

### ISSUE THREAD

**Issue by demontamer** [enigmampc/catalyst#159]
Dear Catalyst Maintainers,

Before I tell you about my issue, let me describe my environment:

# Environment

* Operating System: Ubuntu 16.04
* Python Version: `2.7`
* Python Bitness: `'`
* How did you install Catalyst: (`pip`,)
* Python packages: `$ pip freeze` 

Now that you know a little about me, let me tell you about the issue I am
having:

# Description of Issue
data.history(coin,bar_count=context.window,fields=['open','high','low','close','volume'],frequency='7d')
doesn't work in live mode.
* What did you expect to happen?
Consistent with simulation. Both frequency='1d' and ...'7d' work in backtests
* What happened instead?
'7d' error string printed as exception

Here is how you can reproduce this issue on your machine:

## Reproduction Steps
try resampling price data using 
1. bittrex in live mode
2. data.history(coin,bar_count=context.window,fields=['open','high','low','close','volume'],frequency='7d')
3. Couldn't recover from this issue

Sincerely,
`demontamer

**Comment by fredfortier** [comment]
Currently, we are delegating the creation of OHLCV candles to each exchange API in live mode. This is different from the backtest mode where we are resampling from minute or daily candles. The "7d" frequency is not supported by Bittrex. We'll improve the error message to make this explicit when someone encounters the same issue. We'll also limit the backtest to the frequencies supported by each exchange to avoid differences between backtest and live algos. We may add a resampling feature to the live mode at a later point.

Here are the supported frequencies:
```
            'timeframes': {
                '1m': 'oneMin',
                '5m': 'fiveMin',
                '30m': 'thirtyMin',
                '1h': 'hour',
                '1d': 'day',
            },
```

**Comment by fredfortier** [comment]
Fixed on 0.4.6 please re-open if you still notice an issue.
